webpack/modules/raml-validator-loader/lib/raml-validator-loader.js

```diff
@@ -80,7 +80,7 @@ module.exports =
 	  // Override the default filesystem resolver in order to:
 	  //
 	  // 1) Provide a custom content for the source file
-	  // 2) Track dependent files in order for webpack to invalidate caches
+	  // 2) Track dependant files in order for webpack to invalidate caches
```

Comment:
Hey @mlunoe . The spelling errors were fixed on the clone of the project in DC/OS and not in the project itself. I fixed the spelling errors but on the next version. Please let's merge this in order for me to import the newer version.

Btw, now we have a separate project: https://github.com/dcos/raml-validator-loader after I have properly implemented Jenkins I will publish this module on NPM (we got an approval) and we will forget about such PRs.